Makefile

```diff
@@ -26,7 +27,6 @@ cross: \
 	_output/bin/linux/s390x/checkpoint
 
 release: \
-	clean \
```

Comment:
> Doesn't removing this prevent `clean` from running on all releases?

Yup, and that avoids a parralel-`make` race.  You can always call `make clean` first, but I don't think we need to (more on why not in 3136f293a).

Reply:
The intent here is to always ensure `release` is running in a clean workspace. We should keep this dependency, though calling `make clean release` is redundant in the Travis script.

I'm confused about why this would cause a race. Dependencies always resolve before the block executes, so `release` will always wait on `clean` to finish.

Reply:
> The intent here is to always ensure release is running in a `clean` workspace.

I think the `release` prereq tree is sufficient for this, but if you want to be extra safe we can drop some/all of 3136f293.  Let me know how you want that handled.

> I'm confused about why this would cause a race. Dependencies always resolve before the block executes, so `release` will always wait on `clean` to finish.

Right, but [`release` has no recipe][1].  All of its prereqs will finish before that no-op recipe, but `make` makes no guarantees about the order in which the prereqs are resolved.  For an example using `sleep` to slant the race odds the other way, see the 9e18c3841 commit message.

[1]: https://github.com/kubernetes-incubator/bootkube/blob/8f601b514f74defb31e766be895428012db35cd9/Makefile#L27-L31

Reply:
Ah I see.